library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -80,6 +80,18 @@ public void initialize(FragmentManager fragmentManager, AirMapInterface mapInter
    * @param fragmentManager required for initialization
    */
   public void initialize(FragmentManager fragmentManager) {
+    initialize(fragmentManager, null, null);
+  }
+
+  /**
+   * Used for initialization of the underlying map provider. Uses the default preferred map provider
+   * (currently Native Google Maps, then Mapbox Web maps, then Web Google Maps).
+   *
+   * @param fragmentManager required for initialization
+   * @param mapboxAccessToken Mapbox Access Token (required for Mapbox maps)
+   * @param mapboxMapId Mapbox Map Id (required for Mapbox maps)
+   */
+  public void initialize(FragmentManager fragmentManager, String mapboxAccessToken, String mapboxMapId) {
```

Comment:
Was able to get the `meta-data` approach working.  This allows `DefaultAirMapViewBuilder` to be the run time decision maker as to which Web map provider to use based on the existence of credentials in the app's `AndroidManifest.xml` and provide map provider specific `AirMapViewBuilder` objects to do the building.